ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/ant/tasks/main.yml

```diff
@@ -29,7 +29,9 @@
   tags: ant
 
 - name: Download Apache Ant binaries (macOS) and (Solaris)
-  command: wget https://archive.apache.org/dist/ant/binaries/apache-ant-1.10.5-bin.zip -O /tmp/apache-ant-1.10.5-bin.zip
+  get_url:
```

Comment:
I think in other scripts we're extracting that version

Reply:
Sorry, I did not get the question here. I just changed `wget command` to `get_url` module which help prevent wget not found error on some hosts. I did not touch version.

Reply:
Sorry, extracting 1.10.5 to a single const at the top (assuming it gets repeated)

Reply:
@karianna that is done. Now all hardcoded `1.10.5` replaced by variable `ant_version`